### PR TITLE
Fix iframe HTML structure

### PR DESCRIPTION
**Pull request by palewire**
Per the validator. 
Screenshot from 2022-01-09 08-45-31

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -141,8 +141,7 @@ This is a calendar of regular online events.  It might not be exhaustive.
           style="border:solid 1px #777"
           width="800"
           height="600"
-          frameborder="0"
-          scrolling="no"></iframe>
+          style="border: 0; scrolling: no;"></iframe>
```
Comment (choldgraf) [inline community.md:144]: according to the error message shouldn't we also remove scrolling?

Comment (krassowski) [reply community.md:144]: Yes, this should be replaced by `overflow: hidden`. Also this introduces two `style` attributes - these should be merged.

Comment (palewire) [reply community.md]: Those bugs have been resolved. Thanks for the pointers.

**Comment by palewire** [comment]
I believe your changes are in.

**Review by krassowski — APPROVED** [review]
Thanks! Also, it's much better without the border indeed.

Before:

Screenshot from 2022-01-09 21-45-32

After:

Screenshot from 2022-01-09 21-45-19

**Pull request merged**